### ISSUE TITLE
[ISSUE] DEPRECATED FALSE-POSITIVES

### ISSUE THREAD

**Issue by iWARR** [earthlng/FFprefs-diffs#4]
I propose do not mix "Removed" and "Hidden"
This may confuse a user that makes a deprecated lists comparisons.

At the current representation we can find a prefs that were marked as "Removed", but in fact, they existing at FF75 as [HIDDEN]. This is not right (IMO).

I've check the FF60-FF75 deprecated parts and the next prefs are FALSE-POSITIVE :

```
pref("general.warnOnAboutConfig", true);				[FF71] // FALSE-POSITIVE : EXISTING IN FF75 // [POL] // [HIDDEN]
pref("browser.newtabpage.activity-stream.migrationExpired", false);	[FF67] // FALSE-POSITIVE : EXISTING IN FF75 // [POL] // [HIDDEN]
pref("dom.vr.process.enabled", false);					[FF66] // FALSE-POSITIVE : EXISTING IN FF75
pref("gfx.webrender.all", false);					[FF62] // FALSE-POSITIVE : EXISTING IN FF75 
pref("gfx.webrender.enabled", false);					[FF62] // FALSE-POSITIVE : EXISTING IN FF75
```

For example, you can move these from "Removed" parts to the "Changed" parts of each correspondent FF version, plus mark them as // [HIDDEN] at the end of the line. And that's will be OK.

**To summarize:** If the pref is alive until today, putting it to the "Removed" part is a wrong idea.
Doesn't matter whether pref was hidden or not.

Thanks.

**Comment by earthlng** [comment]
> At the current representation we can find a prefs that were marked as "Removed", but in fact, they existing at FF75 as [HIDDEN]. This is not right (IMO).

Well there's no way for my script to determine if a pref got removed or hidden (or renamed). It can only determine that a pref was visible in one version but not the next and that's why they're all listed under **removed, renamed or hidden**.

But the prefs listed there cannot just be taken at face value as definitely removed or hidden because sometimes they add a pref in 1 version, then make it hidden in another and maybe later make it visible (unhide it) again. For example that's exactly what happened with `gfx.webrender.all` -> originally [added in 59](https://github.com/earthlng/FFprefs-diffs/blob/5562958cb9eed5818e382313c0594565a2bd0def/diffs/5x/diff-v58.0-vs-v59.0.log.js#L50), then [hidden in 62](https://github.com/earthlng/FFprefs-diffs/blob/65df04dd48f975073bfbb9fc48ceeb344e51ae33/diffs/6x/diff-v61.0-vs-v62.0.log.js#L89) and [unhidden in 66](https://github.com/earthlng/FFprefs-diffs/blob/8bbf109724743ca80e9a322031f0477ac754693f/diffs/6x/diff-v65.0-vs-v66.0.log.js#L35).

My diffs list differences in **prefs with default values** from one version to the next and I'm not gonna change anything about that, sorry.

**Comment by iWARR** [comment]
A small clarification. I didn't mean "script" changes, but published diff-lists.
Of course, this is a "manual" one-time fixes over already produced lists.

Or, may be, a good option is having the one separate list of the most important known hidden prefs, still alive for the recent FF version. Just one alive-list of hiddens you may update time to time.

Then a newcomer can make his own "all-time deprecated list" from the all diff versions, exclude hiddens (why not to use a script for that also) and, finally, launch a main cleaning comparison.

P.S. It's just my feedback based on my own experience. Not an issue itself, but a time-saver for others. I'm, personally, don't need this much, because I'm alredy done my general cleaning (thankfully to your kind advises and all friends from "ghacks-user.js").
